static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx

```diff
@@ -126,22 +72,6 @@ export function SdkDocumentation({
 
   const {default: docs} = module;
 
-  if (isFunctionalComponent(docs)) {
```

Comment:
@ArthurKnaus why did we introduce this logic and why are we removing it?

Reply:
In a previous version the docs files exported a component that rendered the onboarding docs.
We gradually migrated them to return the docs as an object to decouple the content from how they are rendered in different scenarios.

This migration is complete so we can remove the old logic branch (that handles the component version).